src/snowflake/snowpark/modin/pandas/dataframe.py

```diff
@@ -3125,33 +3123,37 @@ def __dataframe__(self, nan_as_null: bool = False, allow_copy: bool = True):
             A dataframe object following the dataframe protocol specification.
         """
         # TODO: SNOW-1063346: Modin upgrade - modin.pandas.DataFrame functions
-        ErrorMessage.not_implemented()  # pragma: no cover
+        ErrorMessage.not_implemented(
+            "Snowpark pandas does not support the DataFrame interchange "
+            + "protocol method `__dataframe__`. To use Snowpark pandas "
+            + "DataFrames with third-party libraries like plotly, seaborn, and "
```

Comment:
I don't think you need to mention the specific libraries here, and many of these libraries *just work* because of the __array__ function, regardless of whether the dataframe interchange protocol is supported.

Reply:
Done.